src/helpers/Object.assign.js

```diff
@@ -0,0 +1,28 @@
+//polyfill from https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
+function assign(target) {
+  'use strict';
+  if (target === undefined || target === null) {
+    throw new TypeError('Cannot convert first argument to object');
+  }
+
+  var to = Object(target);
+  for (var i = 1; i < arguments.length; i++) {
+    var nextSource = arguments[i];
```

Comment:
You only really need a simple mixin function, not the whole polyfill here. This whole module could simply be the following.

```js
'use strict';
module.exports = function (target) {
  for (var i = 1; i < arguments.length; i++) {
    var obj = arguments[i];
    for (var prop in obj) {
      if (Object.prototype.hasOwnProperty.call(obj, prop)) {
        target[prop] = obj[prop];
      }
    }
  }
  return target;
};
```

This should also noticeably cut down on the boilerplate.